crates/telio-wg/src/wg.rs

```diff
@@ -817,6 +820,43 @@ impl State {
 
         Ok(success)
     }
+
+    /// This is like the normal '==' but ignores the time_since_last_rx field
+    fn is_same_interface(lhs: &Interface, rhs: &Interface) -> bool {
+        fn is_same_peer(lhs: &Peer, rhs: &Peer) -> bool {
+            lhs.public_key == rhs.public_key
+                && lhs.endpoint == rhs.endpoint
+                && lhs.ip_addresses == rhs.ip_addresses
+                && lhs.persistent_keepalive_interval == rhs.persistent_keepalive_interval
+                && lhs.allowed_ips == rhs.allowed_ips
+                && lhs.rx_bytes == rhs.rx_bytes
+                // this is omitted by design: && lhs.time_since_last_rx == rhs.time_since_last_rx
```

Comment:
why is that?

Reply:
Since we use monotonic clock ([Instant](https://doc.rust-lang.org/std/time/struct.Instant.html#)) to compute the duration which is stored in this field, each time we try to get the current value will will **always** get a different bigger value. A playground [example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=841f64816fd976e057bdd6db976087df).

Reply:
Ah, now I see. Before I tought it was set somewhere in lower lever and just stored in the variable, which would later be used in `update` function.